platform_versioned_docs/version-24.3/getting-started/studios.mdx

```diff
@@ -0,0 +1,452 @@
+        io.save(aligned_pdb_path)
+        
+        return aligned_pdb_path
+        ```
+
+    1. Create a view for a single structure: 
+
+        ```python 
+        def create_structure_view(pdb_path, color, width=400, height=400, label=None):
+            """Create a view for a single structure"""
+            view = py3Dmol.view(width=width, height=height)
+            
+            with open(pdb_path, 'r') as f:
+                pdb_data = f.read()
+            view.addModel(pdb_data, "pdb")
+            view.setStyle({'model': -1}, {'cartoon': {'color': color}})
+            view.zoomTo()
+            
+            if label:
+                view.addLabel(label, {
+                    'position': {'x': 0, 'y': 0, 'z': 0},
+                    'backgroundColor': color,
+                    'fontColor': 'white'
+                })
+            
+            return view
+        ```
+
+    1. Create individual and combined structure views: 
+
+        ```python 
+        def visualize_structures(pdb1_path, pdb2_path):
+            # Align the second structure to the first
+            aligned_pdb2_path = align_structures(pdb1_path, pdb2_path)
+            
+            # Create three separate views
+            view1 = create_structure_view(pdb1_path, 'blue', label="AlphaFold2")
+            view2 = create_structure_view(aligned_pdb2_path, 'darkgrey', label="ESMFold")
+            
+            # Create combined view
+            view3 = py3Dmol.view(width=800, height=400)
+            
+            # Load and display first structure (AlphaFold2)
+            with open(pdb1_path, 'r') as f:
+                pdb1_data = f.read()
+            view3.addModel(pdb1_data, "pdb")
+            view3.setStyle({'model': -1}, {'cartoon': {'color': 'blue'}})
+            
+            # Load and display aligned second structure (ESMFold)
+            with open(aligned_pdb2_path, 'r') as f:
+                pdb2_data = f.read()
+            view3.addModel(pdb2_data, "pdb")
+            view3.setStyle({'model': 1}, {'cartoon': {'color': 'darkgrey'}})
+            
+            # Set up the combined view
+            view3.zoomTo()
+            
+            # Add labels for combined view
+            view3.addLabel("AlphaFold2", {'position': {'x': -20, 'y': 0, 'z': 0}, 'backgroundColor': 'blue', 'fontColor': 'white'})
+            view3.addLabel("ESMFold", {'position': {'x': 20, 'y': 0, 'z': 0}, 'backgroundColor': 'darkgrey', 'fontColor': 'white'})
+            
+            return view1, view2, view3        
+        ```
+
+    1. Display interactive 3D structure views: 
+
+        ```python 
+        # Visualize the structures
+        view1, view2, view3 = visualize_structures(alphafold2_multimer_standard, esmfold_multimer)
+
+        # Display all views
+        print("AlphaFold2 Structure:")
+        view1.show()
+        print("\nESMFold Structure:")
+        view2.show()
+        print("\nAligned Structures:")
+        view3.show()        
```

Comment:
There's no `if __name__ == "__main__":` control block at the bottom. Omitted on purpose or by accident?

Reply:
@FloWuenne can you weigh in? FWIW, the script here worked in both Florian's and my Studios.

Reply:
@gwright99 I suspect it's left out here because we expect users to only run this script directly in their .ipynb, so for our purposes, it's not strictly necessary.